spec/dummy/config/database.travis.yml

```diff
@@ -0,0 +1,7 @@
+# In travis-ci this file is copied over the database.yml before the tests are run.
+# As per https://docs.travis-ci.com/user/database-setup/#SQLite3 if you use activerecord they recommend adding the
+# following configuration for your test database.
+test:
```

Comment:
Just checking this file is still needed based on the current details in `.travis.yml`. It seems to be setting up the CI environment for using Postgres and we are not calling the command to overwrite `database.yml` with this. So would it b clearer to just delete this file now?